deploy/charts/trust-manager/templates/webhook.yaml

```
@@ -83,11 +91,15 @@ metadata:
   labels:
     app: {{ include "trust-manager.name" . }}
     {{- include "trust-manager.labels" . | nindent 4 }}
-{{ if not .Values.app.webhook.tls.helmCert.enabled }}
+  {{- if or (not .Values.app.webhook.tls.helmCert.enabled) .Values.commonAnnotations }}
   annotations:
+    {{- if not .Values.app.webhook.tls.helmCert.enabled }}
     cert-manager.io/inject-ca-from: "{{ include "trust-manager.namespace" . }}/{{ include "trust-manager.name" . }}"
-{{ end }}
-
+    {{- end }}
+    {{- with .Values.commonAnnotations }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
```

```suggestion
    {{- .Values.commonAnnotations | toYaml | indent 4 }}
```

Nit: I am pretty sure this will give the same result, and avoid one control loop.

---

`with` version is safer in case commonAnnotations are empty as it will produce {}